Documentation/QRCodeSign.md

```diff
@@ -0,0 +1,28 @@
+**Instructions**
+
+ All flowers are labeled, allowing visitors to take note of top performing cultivars under regional conditions.
+ 
+ <br>
+ 
+**Search for your favorite flower**
+
+ There are three ways to get to the flower you are looking for:
```

Comment:
Are the following three numbers really ways to get to the flower or is it one way and that is how to get to it. I am not following the wording well here.